aries-site/src/examples/components/layouts/LayerExample.js

```diff
@@ -1,15 +1,14 @@
 import React, { useContext, useState } from 'react';
 import { Button, Box, Heading, Layer, ResponsiveContext, Text } from 'grommet';
 import { Close } from 'grommet-icons';
-import { UsageExample } from '../../../layouts';
 
 export const LayerExample = () => {
   const [open, setOpen] = useState(false);
   const size = useContext(ResponsiveContext);
   const onOpen = () => setOpen(true);
   const onClose = () => setOpen(undefined);
   return (
-    <UsageExample>
+    <Box>
```

Comment:
Could this outer box just be a <> and then the closing would be </> ?

Reply:
Since this is going to be an example demonstrating best practices, I think it is a good idea to implement @taysea 's suggestion.